frontend/js/src/brainzplayer/YoutubePlayer.tsx

```diff
@@ -97,6 +97,13 @@ export default class YoutubePlayer
     return images;
   }
 
+  static getURLFromVideoID(trackID: string): string {
```

Comment:
This would make more sense with the function name:
```suggestion
  static getURLFromVideoID(videoID: string): string {
```